lib/ShellQuote/Any.pm

```diff
@@ -53,7 +53,7 @@ sub _require {
 
 sub _win32_quote {
     my ($cmd) = @_;
-    Win32::ShellQuote::cmd_escape(join ' ', @$cmd);
+    Win32::ShellQuote::quote_system_string(join ' ', @$cmd);
```

Comment:
This should be `Win32::ShellQuote::quote_system_string(@$cmd);`

Reply:
Thanks. Fixed.